Mathlib/Order/Nucleus.lean

```diff
@@ -25,8 +25,8 @@ open Order InfHom Set
 variable {X : Type*}
 
 /-- A nucleus is an inflationary idempotent `inf`-preserving endomorphism of a semilattice.
-In a frame, nuclei correspond to sublocales. -/ -- TODO: Formalise that claim
-structure Nucleus (X : Type*) [SemilatticeInf X] extends InfHom X X where
+In a frame, nuclei correspond to sublocales. -/
+structure Nucleus (X : Type*) [CompleteLattice X] extends InfHom X X where
```

Comment:
@YaelDillies I see you made this change as part of golfing. I gather that probably nobody cares about nuclei on semilattices, but should the docstring be updated here (and below)? It'd proably also be useful to mention `nucleusIsoSublocale` by name.

Reply:
Yes, good point. Let me do that quickly